[12.0] account_move_transactionid_import: Restore feature to import single move

grindtildeath opened the pull request:
TODO:
 - [x] Add tests

Fixing changes done on https://github.com/OCA/account-reconcile/pull/296

grindtildeath committed:
account_move_transactionid_import: Restore feature to import single move

i-vyshnevska commented on account_move_transactionid_import/parser/transactionid_file_parser.py:
```diff
@@ -36,20 +36,19 @@ def __init__(
 
     @classmethod
     def parser_for(cls, parser_name):
         """Used by the new_bank_statement_parser class factory. Return true if
         the providen name is generic_csvxls_transaction
```
hook to check parser .....

grindtildeath replied:
I did removed it altogether since the classes it's inheriting from already return False ;)

gurneyalex committed:
[IMP] account_move_transaction_id_import: single move import

for single move import, put the transactionID in the name of the account move line

adapt the autocompletion rules
